src/components/breadcrumbs/_macro.njk

```diff
@@ -5,7 +5,7 @@
         <ol class="ons-breadcrumbs__items ons-u-fs-s">
             {% for item in params.itemsList %}
                 <li class="ons-breadcrumbs__item{{ ' ' + item.itemClasses if item.itemClasses else '' }}" id="breadcrumb-{{ loop.index }}">
-                    <a class="ons-breadcrumbs__link{{ ' ' + item.linkClasses if item.linkClasses else '' }}" href={% if not isDesignSystemExample %}"{{ item.url }}"{% else %}"#0"{% endif %}{% if item.id %} id="{{ item.id }}"{% endif %}
```

Comment:
This logic shouldn't be reversed. This is now saying if it is a DS example then set the link url to the actual url.